### DISCUSSION TITLE
Next Steps

### DISCUSSION THREAD

**Discussion by mbway** [PyO3/maturin-import-hook#2] [General]
Hi @messense, I was wondering what the next steps are for getting this project deployed. Are you waiting for something from me? I don't mind the wait, I'm just wondering what the situation is.

Let me know if there is anything I can do to help out and I will try to do so if I have time.

**Comment by messense** [comment]
Sorry I simply haven't got the chance to deploy this yet, feel free add pypi release to CI.

**Reply by mbway** [reply]
I have opened a PR to release to pypi https://github.com/PyO3/maturin-import-hook/pull/4

Once that is done, the next things will probably be
- make a release to pypi
- open a PR to maturin to update the documentation about the import hook
- deprecate or remove the old import hook
